include/geode/inspector/criterion/colocation/edgedcurve_colocation.h

```diff
@@ -45,16 +46,12 @@ namespace geode
     public:
         EdgedCurveColocation( const EdgedCurve< dimension >& mesh );
 
-        EdgedCurveColocation(
-            const EdgedCurve< dimension >& mesh, bool verbose );
-
         ~EdgedCurveColocation();
 
         bool mesh_has_colocated_points() const;
 
-        index_t nb_colocated_points() const;
-
-        std::vector< std::vector< index_t > > colocated_points_groups() const;
+        InspectionIssues< std::vector< index_t > >
```

Comment:
I am not sure about these outputs should we have a message to display or not? When do we need this kind of category message?

Reply:
this function feed the big struct.